### DISCUSSION TITLE
Welcome to openmessaging-benchmark Discussions!

### DISCUSSION THREAD

**Discussion by vongosling** [openmessaging/benchmark#183] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by achinjain214** [comment]
openmessaging-benchmark is enabled for ActiveMQ Arrtemis?

i can see a repo having artemis-driver but not sure of that is complete or in working condition?
If someone has already tried it can plz help?
